compiler/rustc_hir_analysis/src/astconv/mod.rs

```diff
@@ -214,7 +214,7 @@ pub struct GenericArgCountResult {
     pub correct: Result<(), GenericArgCountMismatch>,
 }
 
-pub trait CreateSubstsForGenericArgsCtxt<'a, 'tcx> {
+pub trait CreateInstantiationsForGenericArgsCtxt<'a, 'tcx> {
```

Comment:
This is plain wrong, “generic args” would've been the more appropriate term here. This isn't instantiating anything, it's just lowering from HIR GenericArgs to rustc_middle::ty GenericArgs.

The same applies to `SubstsForAstPathCtxt` and `MethodSubstsCtxt`.

However, it's fine to keep this change, [MCP 723](https://github.com/rust-lang/compiler-team/issues/723) / my PR #120926 will update those names anyways.